File: app/src/main/java/at/shockbytes/corey/storage/live/LiveBodyUpdateListener.java
package at.shockbytes.corey.storage.live;

import at.shockbytes.corey.body.goal.Goal;

/**
 * @author Martin Macheiner
 *         Date: 27.02.2017.
 */

public interface LiveBodyUpdateListener {

    void onDesiredWeightChanged(int changed);

    void onBodyGoalAdded(Goal g);

    void onBodyGoalDeleted(Goal g);

    void onBodyGoalChanged(Goal g);

}

File: app/src/main/java/at/shockbytes/corey/body/LiveBodyUpdateListener.kt
package at.shockbytes.corey.body

import at.shockbytes.corey.body.goal.Goal

/**
 * @author Martin Macheiner
 * Date: 27.02.2017.
 */

interface LiveBodyUpdateListener {

    fun onDesiredWeightChanged(changed: Int)

    fun onBodyGoalAdded(g: Goal)

    fun onBodyGoalDeleted(g: Goal)

    fun onBodyGoalChanged(g: Goal)

}
